lib/azure/AzureDataStore.ts

```diff
@@ -70,52 +71,40 @@ export default class AzureDataStore {
    * @param keys
    * @param callback
```

Comment:
Remove. Also low hanging fruits: remove all empty @param. You can see how "fast moving" we were at the time this was written... *facepalm*